src/extensions/averagewaveforms/AverageWaveformPlotNew.tsx

```diff
@@ -123,10 +123,11 @@ const createPlotWaveformLayer = () => {
 
 const HelperPlot = (props: HelperPlotProps) => {
     const plotWaveformLayer = useLayer(createPlotWaveformLayer, props)
+    const layers = useLayers([plotWaveformLayer])
```

Comment:
Turns out it is in fact important to use useLayers for this, otherwise there are too many CanvasWidget renders